### ISSUE TITLE
A better explaination about taming

### ISSUE THREAD

**Issue by Cyberpangolin** [MinetestForFun/server-minetestforfun#519]
When you want to tame an animal / a npc, you have to give it something.

BUT

there is nothing to tell you what to give.

After discussion with @Darcidride , it could be better that right-clicking on the animal/npc says "Not tamed. Give some XXXXX to tame" instead of the simple and useless "Not Tamed."

**Comment by ObaniGemini** [comment]
I don't agree with you. It breaks rp and immersion. As taming in general is a very trivial element of mff's gameplay, I would enjoy, please, if it kept a bit of it's mysterious aspect

**Comment by crabman77** [comment]
It is not so simple, if we change it will be (exemple for kitten)

> "Not tamed. Give some fishing:fish_raw, fishing:clownfish_raw, fishing:bluewhite_raw, fishing:exoticfish_raw, fishing:carp_raw, fishing:perch_raw, fishing:catfish_raw  to tame"

https://github.com/MinetestForFun/server-minetestforfun/blob/master/mods/mobs/kitten.lua#L52

**Comment by Darcidride** [comment]
Myabe wa can do something more RP-like and not so detailled...

Let me think... maybe something like this for kittens : "Not tamed. Your little finger tells you kittens like things from sea, but you forgot what..."
(It will need a new variable from each tamed animal.lua then a little tweak in the API)

**Comment by ObaniGemini** [comment]
Words. Don't make a game like you would make a website or a book. a simple image with a diamond or a fish would be way simpler

**Comment by Darcidride** [comment]
Well, yes an image should be really great, but image in the actual chat... i don't know & think it's actually possible :p

**Comment by ObaniGemini** [comment]
@Darcidride : isn't it possible to have an icon appearing in the center of player's screen, just a bit up to base hud, blinking for 1 second after hitting ?

**Comment by Darcidride** [comment]
@LeMagnesium and @crabman77 are most informed than me for this kind of answer, the fact is the hud is already too heavy in my point of view, it's maybe not a good idea to add this.
Let's wait their answers about this piece of code.
